tasks/get_group_rules.json

```diff
@@ -0,0 +1,5 @@
+{
+  "description": "Run on a PE primary node to return the rules currently applied to the PE Infrastructure Agent group, along with a warning that they will be overwritten",
```

Comment:
Kinda wondering if we should take the warning out of the task and just let it return the rules and output the message from the plans. In case we want to reuse the task. Although we don've have a use at the moment...